packages/prisma/src/prisma/schema/analytics.prisma

```diff
@@ -0,0 +1,17 @@
+model AnalyticsDaily {
+  id Int @id @default(autoincrement())
+
+  timestamp     DateTime @db.Date
+  responseCount Int
+  totalScore    Float
+  totalPoints   Float
+  totalXp       Float
+
+  collectedAchievements String[]
+
+  participant   Participant @relation(fields: [participantId], references: [id])
+  participantId String      @db.Uuid
+
+  course   Course @relation(fields: [courseId], references: [id])
```

Comment:
Should we limit the analytics to be course-specific or should the overview for every user be Klicker-wide?

Reply:
has been updated accordingly